src/app/(with-header)/mypage/activities/components/ActivityCard.tsx

```
@@ -0,0 +1,102 @@
+          <p className='text-2xl font-medium text-gray-900'>
+            ₩{price.toLocaleString()} / 인
+          </p>
+
+          {/* 더보기 옵션 */}
+          <div className='relative'>
+            <button
+              onClick={() => setIsMenuOpen(!isMenuOpen)}
+              className='flex h-40 w-40 items-center justify-center rounded-full hover:bg-gray-100'
+            >
+              <MoreOptionsIcon size={40} />
+            </button>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)